inventory/sample/group_vars/k8s-cluster/k8s-cluster.yml

```diff
@@ -176,6 +176,10 @@ podsecuritypolicy_enabled: false
 volume_cross_zone_attachment: false
 # Add Persistent Volumes Storage Class for corresponding cloud provider ( OpenStack is only supported now )
 persistent_volumes_enabled: false
+
+# Add Pod priority to all the Kube components
+pod_priority_assignment: true
```

Comment:
Does this need to be optional? I think priority classes are always set with kubeadm generated static pod manifests. If we need to make it optional we should do so for kubeadm also.

Reply:
Removed the boolean flag pod_priority-assignment to sync it with kube-adm